README.md

```diff
@@ -381,7 +395,7 @@ To simplify the layout, `DP` is being used as an abbreviation for `DefaultPrompt
 Default prompt layout:
 
 ```text
-{DPPrefix}{DPPath}{PathStatusSeparator}<{BeforeStatus}{Status}{AfterStatus}>{DPBeforeSuffix}<{DPDebug}><{DPTimingFormat}>{DPSuffix}
+{DPPrefix}{BeforePath}{DPPath}{AfterPath}{PathStatusSeparator}<{BeforeStatus}{Status}{AfterStatus}>{DPBeforeSuffix}<{DPDebug}><{DPTimingFormat}>{DPSuffix}
 ```
```

Comment:
Can you also update line 404 below the same way?  Thanks.

Reply:
Done.